README.rst

```diff
@@ -28,9 +32,9 @@
 sciform
 #######
 
-|  **Repository:** `<https://github.com/jagerber48/sciform>`_
-|  **Documentation:** `<https://sciform.readthedocs.io/en/stable/>`_
-|  **PyPi:** `<https://pypi.org/project/sciform/>`_
+| **Repository:** `<https://github.com/jagerber48/sciform>`_
+| **Documentation:** `<https://sciform.readthedocs.io/en/stable/>`_
+| **PyPi:** `<https://pypi.org/project/sciform/>`_
 
```

Comment:
Could you please have a blank line appear in the rendered output between the **PyPi** line and the "We would greatly appreciate..." line?

Reply:
I wish I could, but it seems that GH does not render it the way I think it should. I tried an extra `|` attached or in it's own block, I tried empty lines, etc but GH does not render either. I also don't want to get too complicated just just a blank line in a readme. I think it looks better already and I can't find an easy solution to add a blank line. I suggest to leave as is and live with it - perfect is the enemy of good - and rather put our time into more important things.

Reply:
Ah ok, No problem then. Didn't realize GH is doing something to make it challenging to include a blank line! Ok, I'll look over everything more carefully when I get a chance!